lib/modules/datasource/github-releases/types.ts

```diff
@@ -13,6 +13,7 @@ export type GithubRelease = {
 
 export interface GithubReleaseAsset {
   name: string;
+  url: string;
```

Comment:
why this? can't we reuse the browser download url as in other datasource/ manager? 🤔

Reply:
nah. `browser_download_url` doesn't work with private repository assets. I've tested it